src/canvas_helpers.py

```diff
@@ -318,19 +318,20 @@ def write_tableau_directory(list_of_dfs):
     """
     tableau_path = _make_output_dir("Tableau")
     union = pd.concat(list_of_dfs, axis=0, ignore_index=True)
-    union.to_csv(f"{tableau_path}/module_data.csv", index=False)
+    module_data_output_path = tableau_path / "module_data.csv"
+    union.to_csv(module_data_output_path, index=False)
 
     root = os.path.dirname(os.path.abspath(__file__))[:-4]
 
     # Copy the course_entitlements.csv into the Tableau folder
-    src = root + "/course_entitlements.csv"
-    dst = root + "/data/Tableau/course_entitlements.csv"
+    src = Path(f"{root}/course_entitlements.csv")
+    dst = Path(f"{root}/data/Tableau/course_entitlements.csv")
     shutil.copyfile(src, dst)
 
     current_dt = datetime.datetime.now()
-    dir_name = str(current_dt.strftime("%Y-%m-%d %H:%M:%S"))
+    dir_name = str(current_dt.strftime("%Y-%m-%d %H.%M.%S"))
```

Comment:
I don't think dots are a good convention for file paths? Could this be a dash instead. To avoid any issues with filepaths/parsing. Make sense?

Reply:
That's a good point, I'll adjust so it displays "%Y-%m-%d %H-%M-%S"

Reply:
Changed